spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/ConfigMapConfigProperties.java

```diff
@@ -122,20 +122,21 @@ public String getConfigurationTarget() {
 		private Boolean useNameAsPrefix;
 
 		/**
```

Comment:
nothing has changed here, I just grouped `explicitPrefix` and `includeProfileSpecificSources` to be adjacent